docs/modules/ROOT/pages/user-guide/reports/sankey.adoc

```diff
@@ -0,0 +1,68 @@
+= Sankey Chart
+
+A Sankey visualization will compute a diagram from nodes and links.
+Beware that cyclic dependencies are not supported.
+The Sankey chart will advice you to choose the property to be used on the visualization per Label.
```

Comment:
a comment that applies to the overall feature:

I'm wondering if it makes sense to pass in nodes/relationships directly into the sankey chart. Maybe we should have it use a list of strings like we did with sunburst.

Reasons:
1. In the card footer, we now show the node labels in a given color, but the color isn't really used. It does however let us nicely choose a property value to render as each node, but that could be replaced by a string list.
2. Having a single `value` property determining the thickness of the line might be complicated and require some virtual relationships for complex diagrams. On the other hand,  that's kind of a pattern we have already used. 

Maybe it makes more sense to change the card footer to a string selector, and have users choose the thickness property there instead of the advanced settings. People may not look at the advanced settings in general, get a weird visualization (which happens if there's no value set for the thickness) and get confused.

On the other hand... the string list-based approach introduces another problem, how will we get the thickness for each link? Either way I think we should discuss this together.